src/pages/Settings.vue

```diff
@@ -12,6 +12,14 @@
         <div class="green mt1 absolute save-message" :class="[savedLocale ? 'o-100' : 'o-0']">Saved!</div>
       </div>
     </div>
+    <div class="cf">
```

Comment:
I wonder if this might be better to put into the menu, so it's a bit easier to access. As well, it would let people quickly toggle the theme on the page they're on, to see if they actually like the changes.